### DISCUSSION TITLE
Is there any way to remove scholar.log logging  and overall logging.

### DISCUSSION THREAD

**Discussion by ghost** [scholarly-python-package/scholarly#280] [Q&A]
I have an issue where I can't run the program without sudo command because of logging into the root directory. Is there a way to disable logging from scholarly itself? 

Would very help me :) 
Thank you

**Comment by ipeirotis** [comment]
The latest version of scholarly has logging disabled by default (see issue https://github.com/scholarly-python-package/scholarly/issues/171)

**Reply by ghost** [reply]
it stops logging but still creates the log file. The file is just empty. 

I have reinstalled the scholarly all together and use mac
